setup.py

```diff
@@ -35,6 +35,8 @@
                    'License :: OSI Approved :: BSD License',
                    'Operating System :: OS Independent',
                    'Programming Language :: Python',
+                   'Programming Language :: Python :: 2.7',
```

Comment:
(byte string `b"literals"` (introduced in this PR) were actually backported to Python 2.6, but I think it's worth only advertising support for 2.7 given Python 2's upcoming end-of-life.)

Reply:
Unless it won't actually work, I don't see a good reason to stop advertising support.